bin/celeste.jl

```diff
@@ -75,7 +90,7 @@ function infer(dir, run, camcol, field, outdir, partnum, parts)
     # initialize tiled images and model parameters
     tiled_blob, mp = ModelInit.initialize_celeste(images, cat_entries,
                                                   tile_width=TILE_WIDTH,
-                                                  fit_psf=true)
+                                                  fit_psf=true);
```

Comment:
When debugging, I run these interactively, and it's nice not to have a screenful of output.